frontend/src/components/Footer.vue

```diff
@@ -0,0 +1,99 @@
+<script setup lang="ts"></script>
+
+<template>
+    <div id="footer">
+        <div class="container">
+            <div class="icons">
+                <a class="icon" href="https://fr-fr.facebook.com/BeninExcellence/">
+                    <i class="fa fa-facebook-f" />
+                </a>
+                <a class="icon" href="https://twitter.com/excellencebenin">
+                    <i class="fa fa-twitter" />
+                </a>
+                <a class="icon" href="#i">
```

Comment:
Is it the [same problem](https://github.com/YacineSteeve/quizoot/pull/7/files?diff=unified&w=0#r1016629224) here ?

Reply:
Oh I think I just forgot this one